apps/glusterfs/dbcommon.go

```diff
@@ -227,3 +227,134 @@ func DeleteBricksWithEmptyPath(db *bolt.DB, all bool, clusterIDs []string, nodeI
 	}
 	return nil
 }
+
+func DeletePendingEntries(db *bolt.DB, debug bool) error {
+
+	if debug {
+		logger.SetLevel(utils.LEVEL_DEBUG)
+	}
```

Comment:
It seems a bit odd to me to be setting the logger this way. Normally, I'd want to configure the logger externally to this function. Is the issue that the loggger is "private" to glusterfs package? If so we could just add another function to set the logger (perhaps make setLogLevel in apps/glusterfs/app.go into a standalone exportable function).